launcher/cmd.go

```diff
@@ -51,6 +52,13 @@ var rootCmd = &cobra.Command{
 	Short: "Runs a shortcut",
 	Args:  cobra.NoArgs,
 	Run: func(cmd *cobra.Command, args []string) {
+		var err error
+		CONFIG, err = ParseConfig()
+		if err != nil {
+			fmt.Println(err)
+			os.Exit(1)
+		}
```

Comment:
issue: prefer using `display.Error()` instead of `fmt.Println()` to make sure it's displayed to the desired "CobraDisplay" (GUI or CLI)